packages/@ember/-internals/utils/lib/cache.ts

```diff
@@ -3,15 +3,17 @@ export default class Cache<T, V> {
   public misses = 0;
   public hits = 0;
 
-  constructor(private limit: number, private func: (obj: T) => V, private store?: any) {
-    this.store = store || new Map();
-  }
+  constructor(
+    private limit: number,
+    private func: (obj: T) => V,
+    private store: Map<T, V> = new Map()
```

Comment:
It appears that this initialization form (vs. `|| = new Map()`) is not passing the tests – I will check on why tomorrow!

Reply:
It was actually the possibility of nullish values in the map.